src/shared/scrapers/USA/WV/index.js

```diff
@@ -77,43 +77,85 @@ const scraper = {
     'Wyoming County'
   ],
 
-  async scraper() {
-    const $ = await fetch.page(this.url);
+  scraper: {
+    '0': async function() {
+      const $ = await fetch.page(this.url);
 
-    const $p = $('p:contains("Counties with positive cases")');
+      const $p = $('p:contains("Counties with positive cases")');
 
-    const list = $p
-      .text()
-      .split(':')[1]
-      .split(',');
+      const list = $p
+        .text()
+        .split(':')[1]
+        .split(',');
 
-    let counties = [];
+      let counties = [];
 
-    for (const item of list) {
-      const items = item.split('(');
-      if (items.length !== 2) {
-        continue;
-      }
+      for (const item of list) {
+        const items = item.split('(');
+        if (items.length !== 2) {
+          continue;
+        }
+
+        const county = geography.addCounty(parse.string(items[0]));
+        let cases = items[1];
 
-      const county = geography.addCounty(parse.string(items[0]));
-      let cases = items[1];
+        if (cases.slice(-1) !== ')') {
+          continue;
+        }
 
-      if (cases.slice(-1) !== ')') {
-        continue;
+        cases = parse.number(cases.slice(0, -1));
```

Comment:
fyi parse.number returns 0 if the arg is empty string ... you might want to check that, and put "null" if it's an empty string.  I noted this in another issue.

Reply:
This is for historical cached content. I'd just as soon leave it alone unless you find there's an actual issue.

Reply:
Sounds good!